### PR TITLE
Fix unlisted exceptions in `chronos/apps/http/httpserver.nim`

### PR DESCRIPTION
**Pull request by EmilIvanichkovv**
Fix for #489

### CODE REVIEW

File: chronos/apps/http/httpserver.nim [hunk 1]
```diff
@@ -555,14 +555,16 @@ proc consumeBody*(request: HttpRequestRef): Future[void] {.
   finally:
     await reader.closeWait()
 
-proc getAcceptInfo*(request: HttpRequestRef): Result[AcceptInfo, cstring] =
+proc getAcceptInfo*(request: HttpRequestRef): Result[AcceptInfo, cstring] {.
```
Comment (arnetheduck) [inline chronos/apps/http/httpserver.nim:558]: in general, functions returning `Result` should not raise excpetions - in this particular case, you should trace the call chain down to where the exception is being raised and catch it, translating it to a string instead.

That said, probably you have out-of-date dependencies - this error is not happening in CI

Comment (EmilIvanichkovv) [reply chronos/apps/http/httpserver.nim:558]: > That said, probably you have out-of-date dependencies - this error is not happening in CI
Hmm
I did the following steps:
- git clone 
- nimble install
- nimble setup
- nim c chronos/apps/http/httpserver.nim

Comment (arnetheduck) [reply chronos/apps/http/httpserver.nim:558]: Nimble dependency resolution is broken - you can often get it to work by removing its cache (`rm -rf $HOME/.nimble`) cc @zah

Comment (EmilIvanichkovv) [reply chronos/apps/http/httpserver.nim:558]: Oddly it helped. Ty!

**Pull request closed without merging**